Tools/cases_generator/tier1_generator.py

```diff
@@ -208,7 +255,8 @@ def generate_tier1_from_files(
 ) -> None:
     data = analyze_files(filenames)
     with open(outfilename, "w") as outfile:
-        generate_tier1(filenames, data, outfile, lines)
+        generate_tier1_cases(data, outfile, lines)
```

Comment:
This seems wrong. I don't think this should be changed.

Reply:
`generate_tier1_from_files` is only used as as a test helper.

We need this so that it doesn't dump the prelude and postlude into the test cases in `test_generated_cases`.

Reply:
This is going to break Cinder. Extra files are allowed to allow the bytecode definitions to be spread across several files.
You can strip the extra output off in the `run_cases_test` method.